### PR TITLE
add some missing rocksdb options

### PR DESCRIPTION
**Pull request by Oreoxmt**
Signed-off-by: Aolin <aolin.zhang@pingcap.com>

### What is changed, added or deleted? (Required)

TiDB v7.1.0 introduces two TiKV configuration items: optimize-filters-for-memory and ribbon-filter-above-level.

TiDB v6.2.0 introduces `rocksdb.defaultcf|writecf|lockcf.format-version`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13582
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1368,6 +1374,12 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 + 开启每个 block 建立 bloom filter 的开关。
 + 默认值：false
 
+### `ribbon-filter-above-level` <span class="version-mark">从 v7.1.0 版本开始引入</span>
+
++ 控制是否对于大于等于该值的 level 使用 Ribbon filter，对于小于该值的 level，使用非 block-based bloom filter。当该配置开启时，[`block-based-bloom-filter`](#block-based-bloom-filter) 将被忽略。
++ 只有当时 [`format-version`](#format-version-从-v620-版本开始引入) >= 5 时，该配置项才生效。
```
Comment (hfxsd) [inline tikv-configuration-file.md:1380]:
```suggestion
+ 只有当 [`format-version`](#format-version-从-v620-版本开始引入) >= 5 时，该配置项才生效。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1350,6 +1350,12 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 + `writecf` 默认值：`false`
 + `lockcf` 默认值：`false`
 
+### `optimize-filters-for-memory` <span class="version-mark">从 v7.1.0 版本开始引入</span>
+
++ 控制是否生成能够最小化内存碎片的 Bloom/Ribbon filter。
++ 只有当时 [`format-version`](#format-version-从-v620-版本开始引入) >= 5 时，该配置项才生效。
```
Comment (hfxsd) [inline tikv-configuration-file.md:1356]:
```suggestion
+ 只有当 [`format-version`](#format-version-从-v620-版本开始引入) >= 5 时，该配置项才生效。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1522,6 +1534,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 + 默认值：128MB
 + 单位：KB|MB|GB
 
+### `format-version` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
++ 设置 SST 文件的格式版本。该配置项只影响新写入的表，对于已经存在的表，版本信息会从 footer 中读取。
++ 可选值：
+    - `0`：适用于所有 TiKV 版本。默认 checksum 类型为 CRC32，这个版本不支持修改 checksum 类型。
```
Comment (hfxsd) [inline tikv-configuration-file.md:1541]:
```suggestion
    - `0`：适用于所有 TiKV 版本。默认 checksum 类型为 CRC32。该版本不支持修改 checksum 类型。
```

**Review by hfxsd — APPROVED** [review]

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1522,6 +1534,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
+    - `1`：适用于所有 TiKV 版本。支持使用非默认的 checksum 类型，例如 xxHash。只有在 checksum 类型不是 CRC32 时，RocksDB 才会写入数据。（`0` 版本会自动升级）
+    - `2`：适用于所有 TiKV 版本。更改了压缩块的编码方式，使用 LZ4、BZip2 和 Zlib 压缩。
+    - `3`：适用于 TiKV v2.1 及以上版本。更改了索引块中 key 的编码方式。
+    - `4`：适用于 TiKV v3.0 及以上版本。更改了索引块中 value 的编码方式。
+    - `5`：适用于 TiKV v6.1 及以上版本。全量和分区 filter 采用一种更快、更准确的 Bloom filter 实现，其结构不同。
```
Comment (hfxsd) [inline tikv-configuration-file.md:1546]: 其结构不同  是指什么？可以优化下。

Comment (Oreoxmt) [inline tikv-configuration-file.md:1546]:
```suggestion
    - `5`：适用于 TiKV v6.1 及以上版本。全量和分区 filter 采用一种具有不同架构的更快、更准确的 Bloom filter 实现。
```

**Pull request merged**